Dockerfile

```
@@ -10,11 +10,14 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends \
        sudo systemd systemd-sysv \
        build-essential wget libffi-dev libssl-dev \
-       python3-pip python3-dev python3-setuptools python3-wheel \
+       python3-pip python3-dev python3-setuptools python3-wheel python3-apt \
```

Maintain alphabetical order
```suggestion
       python3-apt python3-pip python3-dev python3-setuptools python3-wheel \
```

---

@dav3r Did you want just that line to be corrected or the entire `apt-get install` command?

---

Ah, I was just looking at that one line, but we might as well fix the whole thing. 👍

---

Do we want to muck with this or just leave it in sync with @geerlingguy's upstream?

---

I don't think the packages change often in his upstream, so I think reformatting to improve readability doesn't make substantial work for future updates.

---

I alphabetized them and broke them onto their own lines in 8285ee6 so it should be straightforward to massage any future updates into place.